components/TextFields/src/private/MDCPaddedLabel.m

```diff
@@ -16,8 +16,23 @@
 
 #import "MDCPaddedLabel.h"
 
+static NSString *const MDCPaddedLabelHorizontalPaddingKey = @"MDCPaddedLabelHorizontalPaddingKey";
+
 @implementation MDCPaddedLabel
 
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super initWithCoder:aDecoder];
+  if (self) {
+    _horizontalPadding = (CGFloat)[aDecoder decodeFloatForKey:MDCPaddedLabelHorizontalPaddingKey];
+  }
+  return self;
+}
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
+  [super encodeWithCoder:aCoder];
+  [aCoder encodeFloat:(float)_horizontalPadding forKey:MDCPaddedLabelHorizontalPaddingKey];
```

Comment:
Should we always be encoding this as a double so we don't lose precision on 64-bit platforms?

Reply:
We could! I took inspiration for float casting it from all the other places we usually do CGFloats. If you look around our codebase, there's a lot of `.f`s to avoid the lost of precision warning. Figured this did just like that.